components/atoms/MoreInfo.js

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+import PropTypes from "prop-types";
+import { faArrowAltCircleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+/**
+ *  More Information Component
+ */
+
+export default function MoreInfo(props) {
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="pb-2 pt-2">
```

Comment:
you can remove the outer div and add mt-4 mb-4 to the div below it has the same effect, just a suggestion it is good also as is

Reply:
Good suggestion, went ahead and did this